engine/src/main/java/com/sandboni/core/engine/Processor.java

```diff
@@ -180,6 +193,15 @@ private Builder getBuilder(Context context) {
         return new Builder(context);
     }
 
+    private Set<String> getAllReflectionCallers(Context context) {
```

Comment:
can't take all edges from REFLECTION_CALL_VERTEX?

Reply:
REFLECTION_CALL_VERTEX will have 3 types of linked vertices (source code, test class, test case). we need only one of these types (test case). 
When I think about it, it will be more efficient to get the edges from REFLECTION_CALL_VERTEX and then filter by link type. will change this. thanks

Reply:
The graph is not constructed yet. I wanted to catch this case before we proceed with any unnecessary actions.

Reply:
we don't have the graph yet (I wanted to catch this case before we do extra work).

Reply:
ok